Extensions/Effects/JsExtension.js

```diff
@@ -125,6 +125,12 @@ module.exports = {
       .setValue('7')
       .setLabel(_('Quality (between 0 and 20)'))
       .setType('number');
+    advancedBloomProperties
+      .getOrCreate('padding')
+      .setValue('0')
+      .setLabel(_('Padding'))
+      .setType('number')
+      .setDescription(_('Padding for filter area'));
```

Comment:
```suggestion
      .setDescription(_('The padding around the effect area'));
```

here and everywhere ("filter" is a PIXI term. In GDevelop, we must always talk about "effect". They are "visual effects", not filters).

Reply:
I replace by 'Padding for visual effects area'

Reply:
`Padding for the visual effect area` then :)